test/index.js

```diff
@@ -59,11 +59,12 @@ test('shows a repo', async t => {
   t.true(agoRegExp.test(await browser.getText('time.updated-at')));
 });
 
-test('checks that the games load up', async () => {
-  await browser.waitForExist('.nav-secret-game');
-  await browser.click('.nav-secret-game');
-  await browser.waitForExist('.secret-game-is-playing', 30 * 1000); // It may take some time to fetch the game from the Gist
-});
+// game functionality has been removed
```

Comment:
If it has been removed completely, why not just remove the whole code? We can't get it working in the future anyway.

For historical purposes, the code will always be there on the git history.

We should also remove the readme section https://github.com/coala/gh-board#easter-eggs

But it said "easter eggs" I'm not sure if it's truly a hidden easter eggs that not very obvious to find, and we miss it. 😂

Reply:
Create an issue regarding removal https://github.com/coala/gh-board/issues/47